core/vm/gas.go

```diff
@@ -35,8 +35,27 @@ var (
 	GasStop   = big.NewInt(0)
 
 	GasContractByte = big.NewInt(200)
+
+	n63 = big.NewInt(63)
+	n64 = big.NewInt(64)
 )
 
+// calcGas returns the actual gas cost of the call.
+//
+// The cost of gas was changed during the homestead price change HF. To allow for EIP150
+// to be implemented. The returned gas is gas - base * 63 / 64.
+func callGas(gasTable params.GasTable, availableGas, base, callCost *big.Int) *big.Int {
+	if gasTable.CreateBySuicide != nil {
+		g := new(big.Int).Sub(availableGas, base)
+		g.Mul(g, n63)
+		g.Div(g, n64)
+		if g.Cmp(callCost) < 0 {
+			return g
+		}
```

Comment:
EIP150 currently says:
```
Define "all but one 64th" of N as N - floor(N / 64) 
``` 
This code implements this as `(N * 63) // 64` which compared to the EIP150 formula can be off-by-one for some gas values: https://play.golang.org/p/QUbnDxCcq_

Either this code or EIP150 needs to change.